### PR TITLE
qemu.tests: Update the commands used for check device inside Windows gue...

### PR DESCRIPTION
**Pull request by ypu**
...st

Make the pci_hotplug tests support Windows guest.

### CODE REVIEW

**Comment by ypu** [comment]
ID: 1081683

File: qemu/tests/cfg/pci_hotplug.cfg [hunk 1]
```diff
@@ -139,8 +141,24 @@
             reference_cmd = lspci
             find_pci_cmd = 'lspci'
             pci_test_cmd = "echo %s; yes | mke2fs `fdisk -l 2>&1 | awk '/\/dev\/[sv]d[a-z]* doesn/ {print $2}'` | sort | tail -1"
+            Windows:
+                reference_cmd = wmic diskdrive list
+                find_pci_cmd = wmic diskdrive list
+                pci_test_cmd = "echo %s &&"
+                pci_test_cmd += " echo select disk 1 > cmd"
```
Comment (xutian) [inline qemu/tests/cfg/pci_hotplug.cfg]: why always select 1 at here? or just a  typo "echo select disk %s > cmd"

Comment (ypu) [inline qemu/tests/cfg/pci_hotplug.cfg]: Thanks for your comment. Update it.

File: qemu/tests/cfg/pci_hotplug.cfg [hunk 2]
```diff
@@ -117,6 +117,8 @@
             only hotplug_block
             pci_model = virtio
             match_string = "Virtio block device"
+            Windows:
+                match_string = "VirtIO SCSI Disk"
```
Comment (suqinhuang) [inline qemu/tests/cfg/pci_hotplug.cfg]: a little strange that "scsi" string is requested in virtio_blk device

Comment (ypu) [inline qemu/tests/cfg/pci_hotplug.cfg]: Yes, but this is what our device register in windows system. Confirmed with feature owner.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by ypu** [comment]
Thanks @xutian @FengYang and @suqinhuang for your review. And will merge it.

**Pull request merged**